cli/testdata/coder_provisioners_jobs_list.golden

```diff
@@ -0,0 +1,3 @@
+ID                                    CREATED AT            STATUS     TAGS                            TYPE                     ORGANIZATION  QUEUE  
+==========[version job ID]==========  ====[timestamp]=====  succeeded  map[owner: scope:organization]  template_version_import  Coder                
+======[workspace build job ID]======  ====[timestamp]=====  succeeded  map[owner: scope:organization]  workspace_build          Coder                
```

Comment:
It would be nice if tags were listed in `k1=v1,k2=v2` format.

Reply:
Good suggestions, and I agree we should hide default owner/scope. We should probably update table formatter to change how maps are output.

Reply:
I would move "Queue" column to be the first one.

Reply:
> I would move "Queue" column to be the first one.

@mtojek I'm going to create a follow-up issue for this as ordering columns is not supported in our table formatter currently. I would like to avoid us having to craft custom structs every time we want to re-order columns.